doc/source/how-to/code/release.yml

```
@@ -39,6 +39,15 @@ jobs:
         run: |
           python -m build && python -m twine check dist/*
 
+      - name: Publish to private PyPI
+        env:
+          REPOSITORY_URL: 'https://pkgs.dev.azure.com/pyansys/_packaging/pyansys/pypi/upload'
+        run: |
+          python -m twine upload dist/* \
+         -u PAT \
+         -p ${{ secrets.PYANSYS_PYPI_PRIVATE_PAT }} \
+         --repository-url $REPOSITORY_URL
+
```

I don't really like exposing this way the PAT... IMO it's better to write it this way (using GH's env variables)
```suggestion
      - name: Publish to private PyPI
        env:
          TWINE_USERNAME: PAT
          TWINE_PASSWORD: ${{ secrets.PYANSYS_PYPI_PRIVATE_PAT }}
          TWINE_REPOSITORY_URL: https://pkgs.dev.azure.com/pyansys/_packaging/pyansys/pypi/upload
        run: |
          pip install twine
          python -m twine upload dist/* \

```

---

I agree! Thanks for the suggestions, @RobPasMue.